### PR TITLE
US03

### PR DESCRIPTION
**Pull request by evabaldwin**
MainModule.feature added, please review. Thanks

### CODE REVIEW

**Comment by DanSpicica** [comment]
Looks good Aynur.
I'll let Sohaila take a look too.

**Comment by evabaldwin** [comment]
Thanks Dan

**Review by Oleksandrozhoh — APPROVED** [review]
clickDashboardModules(String option) - smart way to locate all dashboard modules, great job!

File: src/test/java/net/meetsky/runners/AlulaRunner.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
                 "json:target/cucumber.json"
         },
         publish = true,
-        tags = "@aynur",
+        tags = "",
```
Comment (hamidalakbarov) [inline src/test/java/net/meetsky/runners/AlulaRunner.java:20]: perfect!

**Review by SForogh — APPROVED** [review]
Great job Aynur! Congrats on completing the US. 
A suggestion, what if we use the non-parameterized login method? So, there is no need to pass the value of username and password from Config.properties again... just to avoid repetition.

File: src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java [hunk 2]
```diff
@@ -42,4 +49,19 @@ public void verify_the_user_should_be_at_the_dashboard_page(String title) {
         BrowserUtils.verifyTitleContains(title);
     }
 
+    @And("user logins with valid credentials")
+    public void the_users_log_in_with_valid_credentials() {
+        loginPage.login(ConfigurationReader.getProperty("username"), ConfigurationReader.getProperty("password"));
```
Comment (hamidalakbarov) [inline src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java:54]: Here we can use another login() method that does not requires arguments as it was mentioned by Sohaila

**Pull request merged**